configure.ac

```diff
@@ -457,7 +457,9 @@ AC_C_FLAG([-funwind-tables])
 AC_C_FLAG([-Wall])
 AC_C_FLAG([-Wextra])
 AC_C_FLAG([-Wformat-nonliteral])
-AC_C_FLAG([-Wformat-security])
+AC_C_FLAG([-Wformat=2])
+AC_C_FLAG([-Wtrampolines])
+AC_C_FLAG([-Wbdi-chars=any])
```

Comment:
typo, no such warning exists

Reply:
(there's an `i` missing in `-Wbidi-chars`)

Reply:
I'm removing these for the moment and will add a new PR as that these rae just causing problems